Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -1074,7 +1060,7 @@ - (void)resetTitleView {
     [self animateTitleViewSwapWithNewView:_searchBarContainer
                                completion:^{
                                    self->bResetTitleView = NO;
-                                   [self->activityIndicator stopAnimating];
+                                   [self->_activityIndicator stopAnimating];
```

Comment:
`->` ??? is this still a thing? Any reason for doing this?

Reply:
Fixed, thank you!